samples/dnn/ldm_inpainting.py

```diff
@@ -0,0 +1,457 @@
+import cv2 as cv
+import numpy as np
+import argparse
+from tqdm import tqdm
+from functools import partial
+from copy import deepcopy
+
+backends = (cv.dnn.DNN_BACKEND_DEFAULT, cv.dnn.DNN_BACKEND_INFERENCE_ENGINE,
+            cv.dnn.DNN_BACKEND_OPENCV, cv.dnn.DNN_BACKEND_CUDA)
+targets = (cv.dnn.DNN_TARGET_CPU, cv.dnn.DNN_TARGET_OPENCL, cv.dnn.DNN_TARGET_OPENCL_FP16, cv.dnn.DNN_TARGET_MYRIAD,
+            cv.dnn.DNN_TARGET_HDDL, cv.dnn.DNN_TARGET_CUDA, cv.dnn.DNN_TARGET_CUDA_FP16)
+
+parser = argparse.ArgumentParser(description='Use this script to run inpainting using Latent Diffusion Model',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--encoder', '-e', type=str, help='Path to encoder network.', default=None)
+parser.add_argument('--decoder', '-d', type=str, help='Path to decoder network.', default=None)
+parser.add_argument('--diffusor', '-df', type=str, help='Path to diffusion network.', default=None)
+parser.add_argument('--image', '-i', type=str, help='Path to input image.', default=None)
+parser.add_argument('--samples', '-s', type=int, help='Number of times to sample the model.', default=50)
+parser.add_argument('--backend', choices=backends, default=cv.dnn.DNN_BACKEND_DEFAULT, type=int,
+                        help="Choose one of computation backends: "
+                             "%d: automatically (by default), "
+                             "%d: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                             "%d: OpenCV implementation, "
+                             "%d: CUDA" % backends)
+parser.add_argument('--target', choices=targets, default=cv.dnn.DNN_TARGET_CPU, type=int,
+                        help='Choose one of target computation devices: '
+                             '%d: CPU target (by default), '
+                             '%d: OpenCL, '
+                             '%d: OpenCL fp16 (half-float precision), '
+                             '%d: NCS2 VPU, '
+                             '%d: HDDL VPU, '
+                             '%d: CUDA, '
+                             '%d: CUDA fp16 (half-float preprocess)'% targets)
+
+def make_batch(image, mask):
+    image = image.astype(np.float32)/255.0
+    image = image[None].transpose(0,3,1,2)
```

Comment:
Prefer to use `np.newaxis` to make a new dimension, e.g. `image = image[np.newaxis, ...]`